OneSchedule.Domain/SetWebHookService.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using OneSchedule.Settings;
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Telegram.Bot;
+
+namespace OneSchedule.Domain
+{
+    public class SetWebHookService : IHostedService
```

Comment:
I am not sure that this class should be placed in Domain project. It is mostly related to presentation layer, how app will receive messages. So, I think it's better to move it to OneSchedule project